src/components/Subscribe/Subscribe.tsx

```diff
@@ -73,24 +77,28 @@ const Subscribe = ({
   title = 'Subscribe to Newsletter',
   loadingText = 'Loading...',
   invalidText = '',
+  errorText = 'An error occurred. Please try again.',
```

Comment:
should the `subscriptionState` prop be changed to an object with `{ state: SubscriptionState, message: 'An error occurred', ...}`

This would reduce the number of props and keep us from having to add a different text Prop everytime we needed to add a new state.

Reply:
great idea, but I'll have to change that in Phillips Public as well so it will require more time, can we add another ticker for that? @scottdickerson